migrations.md

```diff
@@ -7,6 +7,105 @@ position: 103
 It is the documentation of the migrations between versions, problems
 encountered while migrating, solutions to problems and changes.
 
+## Nuxt: v3.10.3 👉 v3.12.4
+
+Below you can find a migration checklist;
+
+```markdown
+- [ ] upgrade nuxt: "3.12.4"
+- [ ] upgrade babel/eslint-parser: "7.25.1"
+- [ ] upgrade mermaid-js/mermaid-cli: "10.9.1"
+  - [ ] override puppeteer: "22.15.0"
+- [ ] upgrade nuxt/content: "2.13.2"
+- [ ] upgrade pinia/nuxt: "0.5.3"
+- [ ] upgrade pinia: "2.2.0"
+- [ ] upgrade sass: "1.77.8"
```

Comment:
there is a deprecation error regarding sass upgrade that says move root css rules before any nested. add a note for that as a subtask

note: the error also says wrapping them with `& { ... }` fixes as well, but let's not use it